net.sourceforge.Klavaro.json

```diff
@@ -1,25 +1,56 @@
 {
     "id": "net.sourceforge.Klavaro",
     "runtime": "org.freedesktop.Platform",
-    "runtime-version": "18.08",
+    "runtime-version": "20.08",
     "sdk": "org.freedesktop.Sdk",
     "command": "klavaro",
-    "finish-args":[
+    "finish-args": [
         "--socket=x11",
         "--share=ipc",
         "--socket=wayland",
         "--socket=pulseaudio"
     ],
     "rename-icon": "klavaro",
     "rename-desktop-file": "klavaro.desktop",
-    "modules":[
+    "modules": [
         {
             "name": "adwaita-icon-theme",
+            "cleanup": [
+                "/lib"
+            ],
             "sources": [
                 {
+                    "x-checker-data": {
+                        "type": "gnome",
+                        "name": "adwaita-icon-theme",
+                        "stable-only": true
+                    },
                     "type": "archive",
-                    "url": "http://ftp.gnome.org/pub/GNOME/sources/adwaita-icon-theme/3.32/adwaita-icon-theme-3.32.0.tar.xz",
-                    "sha256": "698db6e407bb987baec736c6a30216dfc0317e3ca2403c7adf3a5aa46c193286"
+                    "url": "https://download.gnome.org/sources/adwaita-icon-theme/3.38/adwaita-icon-theme-3.38.0.tar.xz",
+                    "sha256": "6683a1aaf2430ccd9ea638dd4bfe1002bc92b412050c3dba20e480f979faaf97"
+                }
+            ]
+        },
```

Comment:
This module could presumably now be dropped.

It was added in 88c6e94f139b2f42420ae5f0aa73e8dd8fe78c59:

> Add icon theme until support for host icons is more widespread

I guess host icon support is now widespread.

Reply:
Made this change.